lighthouse-plugin-publisher-ads/audits/duplicate-tags.js

```diff
@@ -78,34 +77,28 @@ class DuplicateTags extends Audit {
   static async audit(artifacts, context) {
     const devtoolsLogs = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
     const networkRecords = await NetworkRecords.request(devtoolsLogs, context);
+    const mainFrameId = networkRecords[1].frameId;
```

Comment:
I think this fails in cases with a redirect. Perhaps we should filter by 200 requests?